tljh/config.py

```diff
@@ -219,11 +219,19 @@ def _is_list(item):
 
 
 def main(argv=None):
+    if os.geteuid() != 0:
+        print("It looks like this command wasn't run with root privileges. Perhaps you didn't use `sudo -E`?")
+        raise(SystemExit)
```

Comment:
Instead of exiting, let's just print this warning. That way, people will see the message *if* they hit an error, but if there is no error, it's okay, too.

```suggestion

```